docs/guide/sitl_setup.rst

```diff
@@ -52,7 +52,7 @@ and Linux (though we recommend the native installations linked above).
 #. Install SSH (Windows only - SSH is present by default on Linux/Mac OSX)
 
    * Download and install `Git for Windows <https://git-scm.com/download/win>`_ (or another client that comes with SSH).
-     After installing you can locate the file using the command ``C:\where ssh`` (normally it is installed to **C:\Program Files (x86)\Git\bin\ssh.exe**
+     After installing you can locate the file using the command ``C:\where ssh`` (normally it is installed to **C:/Program Files (x86)/Git/bin/ssh.exe**
```

Comment:
I'm confused about this change, shouldn't they be backslashes?

Also I believe that can read `where ssh` without the c:\